CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Version
+
+### Added
+- Analysis Reports are now available in html format by specifying the format flag and type html
+- Support has been added for developing other types of formatting options for the Analysis Report
```

Comment:
Why 2 line items for one change? Just "Added support for analysis report to be generated in HTML format" is good enough. You also need to add a link to the PR for the change here.